java/component/keyple-plugin/remotese/src/main/java/org/eclipse/keyple/plugin/remotese/pluginse/VirtualObservableReader.java

```diff
@@ -11,12 +11,12 @@
  ********************************************************************************/
 package org.eclipse.keyple.plugin.remotese.pluginse;
 
-import org.eclipse.keyple.core.seproxy.event.ObservableReader;
+import org.eclipse.keyple.core.seproxy.plugin.ObservableReaderNotifier;
 
 /**
  * Define a Virtual Observable Reader
  */
-public interface VirtualObservableReader extends VirtualReader, ObservableReader {
+public interface VirtualObservableReader extends VirtualReader, ObservableReaderNotifier {
```

Comment:
No, I think that VirtualObservableReader shall extend VirtualReader & ObservableReader
But VirtualObservableReaderImpl has to extend VirtualReaderImpl and to implement VirtualObservableReader & ObservableReaderNotifier